query-engine/query-builders/query-builder/src/lib.rs

```diff
@@ -140,13 +142,59 @@ impl fmt::Display for RelationLink {
 }
 
 #[derive(Debug, Serialize)]
-pub struct DbQuery {
-    pub query: String,
-    pub params: Vec<PrismaValue>,
+#[serde(tag = "type", rename_all = "camelCase")]
```

Comment:
Renaming does not fully work. It turns `TemplateSql` into `templateSql` (which it should not), but fails to turn `placeholder_format` into `placeholderFormat` (which it should).

What's received by TS code is (pretty formatted):
```
{
  "type":"templateSql",
  "fragments": [
    {
      "type":"stringChunk", 
      "value":"DELETE FROM \"public\".\"TestModel\" WHERE 1=1"}],
      "params":[],
      "placeholder_format": {"prefix":"$", "hasNumbering":true}
    }
  ]
}
```

Note that `placeholder_format` has not been renamed.

Reply:
Updating TypeScript code to expect `placeholder_format`, but it is not nice.

Reply:
> It turns TemplateSql into templateSql (which it should not),

It should, this is expected.

> Note that placeholder_format has not been renamed.

This structure doesn't look quite the same at all, here `placeholder_format` is not even part of this structure but a part of `Fragment`.

Reply:
Attempting to put `#[serde(rename_all = "camelCase")]` in front of the `TemplateSql` variant.